v63002/outref/gtm6657.txt

```diff
@@ -0,0 +1,10 @@
+# Disabling Journaling on the Database
+%YDB-I-JNLSTATE, Journaling state for region DEFAULT is now DISABLED
+# Backing up Database with -BKUPDBJNL=OFF
+%YDB-I-JNLSTATE, Journaling state for database file backup is now DISABLED
```

Comment:
This DISABLED output is from the backup command. Additionally, I would like that we copy the backup database onto mumps.dat and do a DSE DUMP -FILE |& $grep "Journal State" to confirm that it is indeed DISABLED and not OFF.